### PR TITLE
LIVE-1407 - Learn page URL & other adjustments

### PR DESCRIPTION
**Pull request by OlivierFreyssinet**
## 🦒 Context (issues, jira)

[LIVE-1407]

## 💻  Description / Demo

- Use new URL for Learn page
- Fix sidebar icon clipped
- Remove outer horizontal padding around Learn iframe (using negative margins)
- Remove title of Learn page (it should be done in the iframe's webpage itself)
- Fix icons & fonts loading in the webpage by adding `allow-same-origin` sandbox attribute on the iframe
- Add developer setting to toggle Learn page URL between prod and staging
- Implement loading / skeleton view
- Handle iframe errors / timeout with an error screen
- Handle "no network" state
- Pass queryParams -> horizontal paddings & light/dark background color

⚠️ this feature is feature flagged for now, it should be there in debug but not in prod. [However there is a testable staging build here, with no feature flagging](https://github.com/LedgerHQ/ledger-live-build/actions/runs/1923099550) made from that commit https://github.com/LedgerHQ/ledger-live-desktop/pull/4777/commits/bbcc37568644e2b0b94abce0fb12141f28f99120

https://user-images.githubusercontent.com/91890529/156396310-27a1b3e6-8194-4b43-a29f-18599780cfb5.mov

## 🖤  Expectations to reach

PR must pass CI, rebase develop if conflicts. Thanks!

- **on QA**: at least one of these two checkboxes must be checked:
  - [ ] a specific test planned is defined on Jira
  - [ ] this PR is covered by automatic UI test
- **on delivery**: at least one of these two checkboxes must be checked: 
  - [x] Option 1: **no impact**: The changes of this PR have ZERO impact on the userland (invisible for users)
  - [x] Option 2: **atomic delivery**: the changes is atomic and complete (no partial delivery)

[LIVE-1407]: https://ledgerhq.atlassian.net/browse/LIVE-1407?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ofreyssinet-ledger** [comment]
Skeleton & offline state implemented

https://user-images.githubusercontent.com/91890529/156220965-77dfadb5-674b-444b-b1b1-364e3e970d39.mov

File: src/renderer/components/SideBar/SideBarListItem.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ export type Props = {
   label: string | (Props => React$Node),
   desc?: Props => any, // TODO: type should be more precise, but, eh ¯\_(ツ)_/¯
   icon?: any, // TODO: type should be more precise, but, eh ¯\_(ツ)_/¯
+  iconSize?: number;
```
Comment (thomasrogerlux) [inline src/renderer/components/SideBar/SideBarListItem.js:14]: Lint error on the `;` (should be `,`)

**Review by thomasrogerlux — APPROVED** [review]
Roses are red
Violets are blue
Why is it a PNG
When it can be an SVG

File: src/renderer/components/SideBar/SideBarListItem.js (hunk 1 above)
Comment (ofreyssinet-ledger) [reply src/renderer/components/SideBar/SideBarListItem.js:14]: ah good eyes thanks!

**Comment by ofreyssinet-ledger** [comment]
@thomasrogerlux I like simple things haha

File: src/renderer/components/SideBar/SideBarListItem.js (hunk 1 above)
Comment (ofreyssinet-ledger) [reply src/renderer/components/SideBar/SideBarListItem.js:14]: my config is so bad that for me everything in Flow is red anyway haha, hard to catch lint errors like this
Screenshot 2022-03-02 at 17 14 50

**Review by LFBarreto — APPROVED** [review]
:+1:

**Pull request merged**